src/sections/Home/Home.tsx

```diff
@@ -26,6 +27,8 @@ export const Home = () => {
         cupiditate, expedita neque eius reiciendis in iusto natus saepe, culpa
         voluptas nostrum, id quo unde odit facilis corporis minima amet.
       </article>
+      <ButtonComponent primary>Primary</ButtonComponent>
```

Comment:
Please refactor all the components you have not to include the word "Component". This makes your code less readable

Reply:
https://github.com/igorizviekov/mt-booking_app/commit/77033a80d298d75278149311ba8b4af03bc0c71d